pex/pip/tool.py

```diff
@@ -232,51 +233,6 @@ def _patch_code(cls, code):
             code_fp.write(code.encode("utf-8"))
         return {"PEX_EXTRA_SYS_PATH": patches_dir, cls._PATCHES_MODULE_ENV_VAR_NAME: patches_module}
 
-    @classmethod
-    def create(
```

Comment:
This went here above: https://github.com/pantsbuild/pex/pull/1892/files#diff-a6a6ecdb319224fc07ed4e7a1a69686efc58ce1501e65eb057e684e907f34a12R32